.travis.yml

```diff
@@ -7,8 +7,11 @@ install:
   - ./scripts/install-protoc.sh
 
 script:
+- test -z $(go fmt ./...) || (echo "Run go fmt ./... and recommit your code";exit 1)
 - go get -u github.com/golang/protobuf/protoc-gen-go
-- make
+- go generate ./...
+- go build ./...
+- go test ./...
 
```

Comment:
#19 merged, so you can rebase on top to remove these now.

Reply:
Woot!  Rebased! :)